packages/components/date-picker/src/date-picker-com/basic-year-table.vue

```diff
@@ -4,29 +4,26 @@
     :aria-label="t('el.datepicker.yearTablePrompt')"
     :class="ns.b()"
     @click="handleYearTableClick"
+    @mousemove="handleMouseMove"
   >
     <tbody ref="tbodyRef">
-      <tr v-for="(_, i) in 3" :key="i">
-        <template v-for="(__, j) in 4" :key="i + '_' + j">
-          <td
-            v-if="i * 4 + j < 10"
-            :ref="
-              (el) =>
-                isSelectedCell(startYear + i * 4 + j) && (currentCellRef = el as HTMLElement)
-            "
-            class="available"
-            :class="getCellKls(startYear + i * 4 + j)"
-            :aria-selected="`${isSelectedCell(startYear + i * 4 + j)}`"
-            :tabindex="isSelectedCell(startYear + i * 4 + j) ? 0 : -1"
-            @keydown.space.prevent.stop="handleYearTableClick"
-            @keydown.enter.prevent.stop="handleYearTableClick"
-          >
-            <div>
-              <span class="cell">{{ startYear + i * 4 + j }}</span>
-            </div>
-          </td>
-          <td v-else />
-        </template>
+      <tr v-for="(row, key) in rows" :key="key">
+        <td
+          v-for="(cell, key_) in row"
+          :key="key_"
```

Comment:
The current writing method will repeat the key.
```suggestion
          :key="key + '_' + key_"
```